bin/rake

```diff
@@ -1,9 +1,4 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path('../spring', __FILE__)
-rescue LoadError => e
-  raise unless e.message.include?('spring')
-end
```

Comment:
I don't think this should've been removed. See https://github.com/rails/spring for an explanation of what spring is. It should be re-added by executing `bundle exec spring binstub --all`